src/oci.c

```diff
@@ -60,6 +60,9 @@
 #include "pod.h"
 #include "namespace.h"
 
```

Comment:
Pertaining to the commit message itself,  I think the docker flow in 1.12.x was the same,(create->start->kill->delete), delete was called if the signal itself was a terminating signal i.e if the shim to be terminated.  I think what has changed is that docker now expects all processes (shim+VM) to be terminated for the delete to be called. At least this was my understanding regarding the change from earlier discussions.